src/main/java/de/retest/web/ScreenshotProvider.java

```diff
@@ -47,11 +50,15 @@ private static BufferedImage shootFullPage( final WebDriver driver ) {
 		final BufferedImage image = Shutterbug
 				.shootPage( driver, ScrollStrategy.WHOLE_PAGE, SCROLL_TIMEOUT_MS, USE_DEVICE_PIXEL_RATIO ).getImage();
 		return driver instanceof ChromeDriver
-				? resizeImage( image, image.getWidth() / SCALE, image.getHeight() / SCALE ) : image;
+				? ImageUtils.resizeImage( image, image.getWidth() / SCALE, image.getHeight() / SCALE ) : image;
```

Comment:
Find it a bit confusing that you directly call `ImageUtils` here, but otherwise via the private method.

Reply:
Done differently anyways...